0800-appendix-198066-thompson.Rmd

```diff
@@ -38,12 +38,16 @@ At crossing `r as.character(my_site)`, `r fpr::fpr_my_pscis_info()` is a `r fpr:
 
 
 <br>
 
-PSCIS crossing 123376 on Highway 16 was assessed by @gollner_etal2013FishPassage in 2013. This is an open bottom structure (pipe arch) that was passable to all life stages at the time of survey. It was 7m in diameter and 24m in length. It is important to note that the majority of flow from Thompson Creek comes from this section that flows under the highway as opposed to the channel mapped as the mainstem. This site was revisited in 2021, with photos shown in `r if(gitbook_on){knitr::asis_output("Figures \\@ref(fig:photo-123377-03) - \\@ref(fig:photo-123377-04).")}else(knitr::asis_output("Figure \\@ref(fig:photo-123377-d02)."))` There was minimal flow with much of the riparian vegetation removed and clear evidence of cattle trampled banks upstream and downstream. There was private land on both sides of the road. Crossings 123381 and 123380 located upstream of the highway were not accessible and could not be surveyed because they were on private land and blocked behind gates. However these sites likely do not have significant influence on connectivity in the watershed because the main body of the stream flows through crossing 123376. 
+PSCIS crossing 123376 on Highway 16 was assessed by @gollner_etal2013FishPassage in 2013. This is an open bottom structure (pipe arch) that was passable to all life stages at the time of survey. It was 7m in diameter and 24m in length. It is important to note that the majority of flow from Thompson Creek comes from this section that flows under the highway as opposed to the channel mapped as the mainstem. This site was revisited in 2021, with photos shown in `r if(gitbook_on){knitr::asis_output("Figures \\@ref(fig:photo-123377-03) - \\@ref(fig:photo-123377-04).")}else(knitr::asis_output("Figure \\@ref(fig:photo-123377-d02)."))` Crossings 123381 and 123380 located upstream of the highway were not accessible and could not be surveyed because they were on private land and blocked behind gates. However these sites likely do not have significant influence on connectivity in the watershed because the main body of the stream flows through crossing 123376. Further upstream, PSCIS crossing 123382 was also visited `r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-123382-01) - \\@ref(fig:photo-123382-02)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-123382-d01))."))` There was minimal flow with much of the riparian vegetation removed and clear evidence of cattle trampled banks upstream and downstream. There was private land on both sides of the road. 

```

Comment:
fyi - things I will  do:

reword the story of the channel re-alignment to give less emphasis to about it flowign to the north. seems sus

specifically state that the mainstem is mapped as crossing the highway at crossing xxxxx. 

explain where specifically  123381 and 123380 are

explain where 123382 is in relation to the other crossings (ex. 4km or whatever upstream and 1.3km or whatever below the outlet of fishpan lake